force-app/main/default/classes/SOQL.cls

```diff
@@ -309,8 +299,7 @@ public with sharing class SOQL {
      * @return List of contacts with the same Mailing State as the Account Billing State
      */
     public static List<Contact> getAccountsWithContactsByState(Id accountId) {
-        // Query the Account
-          Account[] getAccountState = [SELECT Id,Name,BillingState FROM Account WHERE Id = :accountId];
+        Account[] getAccountState = [SELECT Id,Name,BillingState FROM Account WHERE Id = :accountId];
```

Comment:
When querying by Id, you can also go directly to a single object instead of a list